config/locales/interface/slides/en_overview.yml

```diff
@@ -1,76 +1,61 @@
 en:
   slides:
-    hidden_infrastructure_gas_network:
-      title: Gas network
+    scenario_overview:
+      title: Overview
       short_description:
-      description:
-    introduction_to_etm:
-      title: How does the Energy Transition Model work?
+      description: |
+        This section gives a general overview of the current scenario.<br/><br/>
+        When starting a new scenario, this overview shows the current energy system of the selected
```

Comment:
Small difference but I'm not sure that I agree that it shows the 'current' energy system. This Sankey shows the future energy system - when no slides have been touched it is as similar as it can be to the current energy system, but not exactly the same right?